contractcourt/chain_arbitrator.go

```diff
@@ -335,8 +335,8 @@ func (a *arbChannel) NewAnchorResolutions() (*lnwallet.AnchorResolutions,
 // ForceCloseChan should force close the contract that this attendant is
 // watching over. We'll use this when we decide that we need to go to chain. It
 // should in addition tell the switch to remove the corresponding link, such
-// that we won't accept any new updates. The returned summary contains all items
-// needed to eventually resolve all outputs on chain.
+// that we won't accept any new updates. The returned summary will NOT have the
+// resolutions included.
```

Comment:
To make this even more clear, and since we actually only really ever need the `CloseTx` of the summary, we could change this method to only return the close TX.
So just:

```
func (a *arbChannel) ForceCloseChan() (*wire.MsgTx, error) {

...

	closeSummary, err := chanMachine.ForceClose(
		lnwallet.WithExcludeResolutions(),
	)
	if err != nil {
		return nil, err
	}
	
	return closeSummary.CloseTx, nil
```

Would make the mock even simpler as well.

Reply:
great idea